app/renderer/components/Inspector/RecordedActions.js

```diff
@@ -89,7 +101,7 @@ class RecordedActions extends Component {
 
     const highlightedCode = this.code(false);
 
-    return <Card title={<span><Icon type="code-o"/> {t('Recorder')}</span>}
+    return <Card title={<span><LegacyIcon type="code-o"/> {t('Recorder')}</span>}
```

Comment:
in antd documentation it is stated that `Legacy Icon usage will be discarded`. Should we already stop using it?

Reply:
Yes. I'm thinking to work for the LegacyIcon stuff as following PRs since I need to dig them in more